asset-metadata/README.md

```diff
@@ -0,0 +1,54 @@
+## Web Asset Metadata
+ 
+Metadata is a special property on Web asset,
+that will enable extra headers sent from the device to the 
+target page of web asset.
+
+These extra headers mainly include screen related information,
+like hostname, screen name, location, etc.
+
+Example headers
```

Comment:
Example header values? not sure, just need to make sure people understand that the list of headers is complete. But values are an example